src/main/java/com/zufar/icedlatte/order/entity/Order.java

```diff
@@ -0,0 +1,95 @@
+package com.zufar.icedlatte.order.entity;
+
+import com.zufar.icedlatte.openapi.dto.OrderStatus;
+import jakarta.persistence.*;
+import lombok.*;
+
+import java.math.BigDecimal;
+import java.time.OffsetDateTime;
+import java.util.Objects;
+import java.util.Set;
+import java.util.UUID;
+
+@Getter
+@Setter
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name = "orders")
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.UUID)
+    private UUID id;
+
+    @Column(name = "user_id", nullable = false)
+    private UUID userId;
+
+    @Enumerated(EnumType.STRING)
+    @Column(name = "order_status", nullable = false)
+    private OrderStatus status;
+
+    @Column(name = "created_at", nullable = false)
+    private OffsetDateTime createdAt;
+
+    @OneToMany(mappedBy = "order",
+            cascade = {CascadeType.PERSIST, CascadeType.MERGE,
+                    CascadeType.REFRESH, CascadeType.REMOVE, CascadeType.DETACH},
+            orphanRemoval = true,
+            fetch = FetchType.EAGER)
+    private Set<OrderItem> items;
+
+    @Column(name = "items_quantity", nullable = false)
+    private Integer itemsQuantity;
+
+    @Column(name = "total_products_cost", nullable = false)
+    private BigDecimal totalProductsCost; // TODO: should we store this in DB?
+
+    @Column(name = "delivery_cost", nullable = false)
+    private BigDecimal deliveryCost;
+
+    @Column(name = "tax_cost", nullable = false)
+    private BigDecimal taxCost;
+
+    @Column(name = "total_order_cost", nullable = false)
+    private BigDecimal totalOrderCost; // TODO: should we store this in DB?
```

Comment:
no we should not

Reply:
Removed in [46e65b4](https://github.com/Sunagatov/Iced-Latte/pull/217/commits/46e65b4c9646349a131383ace42afdeddba4b663)

Just curious: what about `itemsQuantity`?